src/index.js

```diff
@@ -118,19 +118,19 @@ Calendar.propTypes = {
   lastRenderedDay: dayType.isRequired,
   monthClassName: PropTypes.string,
   monthHeaderClassName: PropTypes.string,
-  monthHeaderFormat: PropTypes.string,
 
   // Function that takes a moment instance and returns a single ReactElement.
   renderDay: PropTypes.func.isRequired,
+  renderMonthHeader: PropTypes.func.isRequired,
   weekClassName: PropTypes.string,
 };
 
 Calendar.defaultProps = {
   compactMonths: false,
   dayAbbrevs: ['Su', 'M', 'T', 'W', 'Th', 'F', 'Sa'],
   dayHeaderStyle: DayHeaderStyles.InFirstMonth,
-  monthHeaderFormat: 'MMMM YYYY',
   renderDay: (day) => <div>{day.format('YYYY-MM-DD')}</div>,
+  renderMonthHeader: (month) => day.format('MMMM YYYY'),
```

Comment:
I was thinking the signature of this function would be `(month, format) => month.format(format)`, and then we'd keep the `monthHeaderFormat` prop for the more general case.